src/utils.cpp

```diff
@@ -489,14 +484,15 @@ MinimizerHitClusters get_minimizer_hit_clusters(
     const std::vector<std::string> &prg_names,
     std::shared_ptr<MinimizerHits> minimizer_hits,
     std::shared_ptr<pangenome::Graph> pangraph, const int max_diff,
-    const uint32_t& genome_size, const float& fraction_kmers_required_for_cluster,
+    const float& fraction_kmers_required_for_cluster,
```

Comment:
what does this variable do?

Reply:
In this function (`get_minimizer_hit_clusters()`), nothing, that is why it was removed.

Reply:
it's green? i mean fraction_kmers_required_for_cluster

Reply:
`fraction_kmers_required_for_cluster` is not a new parameter, the diff is showing it because `genome_size` was removed. `fraction_kmers_required_for_cluster` is, as the name says, the fraction of kmers required to be mapped define a cluster of hits, and defaults to `exp(-e_rate * k)/2` (page 45 in Rachel's thesis)

Reply:
thx!